blocks/api/test/parser.js

```diff
@@ -150,7 +150,7 @@ describe( 'block parser', () => {
 			} );
 
 			const parsed = parse(
-				'<!-- wp:core/test-block smoked:yes -->' +
+				'<!-- wp:core/test-block smoked="yes" url="http://google.com" -->' +
```

Comment:
Yes, handling spaces and special chars like these is one of the reasons I made this switch